src/builtin.cpp

```diff
@@ -1731,15 +1732,13 @@ int builtin_function(parser_t &parser, io_streams_t &streams, const wcstring_lis
                 }
             }
             if (function_name_shadows_builtin && !shadow_builtin) {
-                append_format(
-                    *out_err,
-                    _(L"%ls: function name shadows a builtin so you must use '--shadow-builtin'"),
-                    argv[0]);
-                res = STATUS_BUILTIN_ERROR;
+                debug(1, _(L"%ls \"%ls\" shadows a builtin so you should use '--shadow-builtin' "
```

Comment:
I think a better phrase is "a builtin command "%ls" already exists (use --shadow-builtin to suppress this warning)"